### PR TITLE
add check wildcard characters are contained or not

### PR DESCRIPTION
**Pull request by Konboi**
before this change, if the test_path method receives test path strings which are contained wildcards, this method sets test_paths directly.

```sh
$ launchable subset --build 1 --target 20% minitest "test/**/*.rb"
test/**/*.rb

$ launchable subset --build 1 --target 20% minitest test/**/*.rb
test/application_system_test_case.rb
test/channels/application_cable/connection_test.rb
test/controllers/user_test.rb
test/models/admin/user_test.rb
test/models/open_class_user_test.rb
test/models/user_copy_test.rb
test/models/user_test.rb
test/test_h*ge.rb
test/test_helper.rb
```

so I fixed the test_path method expands wildcards 
```sh
$ launchable subset --build 1 --target 20% minitest "test/**/*.rb"
test/test_h*ge.rb
test/application_system_test_case.rb
test/test_helper.rb
test/models/open_class_user_test.rb
test/models/user_copy_test.rb
test/models/user_test.rb
test/models/admin/user_test.rb
test/controllers/user_test.rb
test/channels/application_cable/connection_test.rb
```

SHELLs usually expand wildcard, but when we combine some commands, SHELLs realize the wildcards as strings and don't expand, I think

### CODE REVIEW

File: launchable/commands/subset.py [hunk 1]
```diff
@@ -116,6 +116,11 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            if any(s in path for s in ('/**', '**/', '*.')):
```
Comment (Konboi) [inline launchable/commands/subset.py:119]: Are there other cases?

File: launchable/commands/subset.py [hunk 2]
```diff
@@ -116,6 +116,11 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            if isinstance(path, str) and any(s in path for s in ('/**', '**/', '*.')):
```
Comment (ninjinkun) [inline launchable/commands/subset.py:119]: Do we need to consider `*`, `/*`case?

Comment (Konboi) [reply launchable/commands/subset.py:119]: if add `*`, `test/test_h*ge.rb` match this if statement and happen infinity loop error.
so I'll add only `/*`.

(Ofcource, I know `test/test_h*ge.rb`is rare case)

Comment (Konboi) [reply launchable/commands/subset.py]: @ninjinkun I added https://github.com/launchableinc/cli/pull/188/commits/2e2b75762e12131e9dcf468d09d58fbcb7391fb9

**Review by ninjinkun — APPROVED** [review]
Looks good!

File: launchable/commands/subset.py (hunk 2 above)
Comment (kohsuke) [reply launchable/commands/subset.py]: Help me understand why this is not `s in path for s in ('*', '?')` and instead there are restrictions about where the wildcard appears. This feels like a rough edge waiting to bite somebody. I can totally imagine somebody doing something like `foo/*/*_test.py` somewhere.

Why would this result in an infinite loop? And even if it does, shouldn't we resolve that by not doing recursion?

Comment (Konboi) [reply launchable/commands/subset.py]: thank you. your point is right.

before I added this change (https:/github.comlaunchableincclipull188commits521288834e1f0c26ee6fdd512b6fcd58027704b6), an infinite loop error had happened.
but now, we checked the path is a directory or not, so it doesn't happen.

I'll fix it.

**Pull request merged**